gymnasium/gymstatic
# Duplicating events content as “Webinars”

- All events content is duplicated as `Webinars`.  Files, paths, and collections are all duplicated (for now).

# To test:

1.  test landing page
    - visit [`/static/events`](https://deploy-preview-54--gymcms.netlify.com/static/events/), you should see "events" show up
    - visit [`/static/webinars`](https://deploy-preview-54--gymcms.netlify.com/static/webinars/) - it should be the same content, but with "events" replaced with "webinars".
1. test event page
    - visit [`/static/events/web_design_is_hard/`](https://deploy-preview-54--gymcms.netlify.com/static/events/web_design_is_hard/), to see the event page version of the Web Design is Hard webinar page
    - now visit [`/static/webinars/web_design_is_hard/`](https://deploy-preview-54--gymcms.netlify.com/static/webinars/web_design_is_hard/) to see it listed under the webinars list.
